src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ValueDistFunction.java

```diff
@@ -70,8 +69,7 @@ public double getMaxDuration() {
 	public double calculateDistanceLocation(LatLng interest) throws Exception {
```

Comment:
Serait-il possible de préciser dans la doc le cas où une exception est soulevée

Reply:
Fixed in commit ea9579e